src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs

```diff
@@ -290,6 +290,29 @@ public async Task<IActionResult> ConfirmDetailsAndSendRequestPage(ConfirmDetails
             }
         }
 
+        [HttpGet]
+        [Route("{apprenticeshipHashedId}/change-provider/cancel", Name = RouteNames.CancelChangeOfProviderRequest)]
+        public async Task<IActionResult> CancelChangeOfProviderRequest(ChangeOfProviderRequest request)
+        {
+            var viewModel = await _modelMapper.Map<CancelChangeOfProviderRequestViewModel>(request);
+
+            return View(viewModel);
+        }
+
+        [HttpPost]
+        [Route("{apprenticeshipHashedId}/change-provider/cancel", Name = RouteNames.CancelChangeOfProviderRequest)]
+        public async Task<IActionResult> CancelChangeOfProviderRequest(CancelChangeOfProviderRequestViewModel viewModel)
+        {
+            if (viewModel.CancelRequest.Value)
+            {
+                return RedirectToRoute(RouteNames.ApprenticeDetail, new { ApprenticeshipHashedId = viewModel.ApprenticeshipHashedId, AccountHashedId = viewModel.AccountHashedId });

```

Comment:
should this be something like 
return Redirect(_linkGenerator.ApprenticeDetails(request.AccountHashedId, request.ApprenticeshipHashedId));

As apprentice details page is currently in V1